backend/src/main/java/cloudflight/integra/backend/controller/AuthController.java

```
@@ -60,4 +61,22 @@ public ResponseEntity<HttpStatus> login(@RequestBody User user) {
                     .build();
         }
     }
+
+    @GetMapping("/me")
+    public ResponseEntity<?> getCurrentUser(@RequestHeader(HttpHeaders.AUTHORIZATION) String authHeader) {
+        if (authHeader == null || !authHeader.startsWith("Bearer ")) {
+            return ResponseEntity.status(HttpStatus.UNAUTHORIZED)
+                    .build();
+        }
```

This is redundant.
Should fail early because of the JWTFilter